docs/source/write_first_app.rst

```diff
@@ -101,889 +97,485 @@ bringing up a new one.
-        ID: id,
-        Color: color,
-        Size: size,
-        Owner: owner,
-        AppraisedValue: appraisedValue,
-    };
-    return ctx.stub.putState(id, Buffer.from(JSON.stringify(asset)));
-  }
-
-Terminal output:
-
-.. code:: bash
-
-  Submit Transaction: CreateAsset, creates new asset with ID, color, owner, size, and appraisedValue arguments
+  await contract.submitTransaction(
+      'CreateAsset',
+      assetId,
+      'yellow',
+      '5',
+      'Tom',
+      '1300',
+  );
 
 .. note:: In the application and chaincode snippets above, it is important
-          to note that the sample application submits the ``'CreateAsset'``
+          to note that the sample application submits the ``CreateAsset``
           transaction with the same type and number of arguments the chaincode
-          is expecting, and in the correct sequence. In this case, the transaction
-          name and correctly sequenced arguments are: ``'CreateAsset'``, ``'asset13'``,
-          ``'yellow'``, ``'5'``, ``'Tom'``, ``'1300'`` because the corresponding chaincode
-          CreateAsset is expecting the correct sequence and type of arguments that define the asset object:
-          sequence: ID, Color, Size, Owner, and AppraisedValue
-
-          type: ID (string), Color (string), Size (int), Owner (string), AppraisedValue (int).
-
-The sample application then evaluates a query for 'asset13'.
-
-Sample application ``'ReadAsset'`` call
-
-  .. code:: bash
-
-    console.log('\n--> Evaluate Transaction: ReadAsset, function returns an asset with a given assetID');
-    result = await contract.evaluateTransaction('ReadAsset', 'asset13');
-    console.log(`*** Result: ${prettyJSONString(result.toString())}`);
-
-Chaincode ``'ReadAsset'`` function
-
-  .. code:: bash
+          is expecting, and in the correct sequence. In this case the correctly sequenced arguments are:
+          
+          .. code-block:: text
+          
+            assetId, "yellow", "5", "Tom", "1300"
+          
+          because the corresponding chaincode ``CreateAsset`` is expecting the following sequence of arguments that
+          define the asset object:
+          
+          .. code-block:: text
+
+            ID, Color, Size, Owner, AppraisedValue
+          
+          Similar to transaction function return values, any data type can be converted to bytes and passed as
+          transaction arguments, provided those arguments match what is expected by the transaction function. Since
+          strings are commonly used, the application API accepts UTF-8 strings as transaction arguments and converts
+          them to bytes internally.
+
+Next, the sample application submits a transaction to transfer ownership of the newly created asset. This time
+the transaction is invoked using ``submitAsync()``, which returns after successfully submitting the endorsed
+transaction to the ordering service instead of blocking until the transaction is committed to the ledger. This allows
```

Comment:
ok, what is blocking. packaging in a block perhaps.

Reply:
https://stackoverflow.com/questions/2407589/what-does-the-term-blocking-mean-in-programming

Reply:
Changed the wording to make it clearer for non-programmers.